src/geonovum/informative.js

```diff
@@ -0,0 +1,13 @@
+// Module w3c/informative
```

Comment:
Let's fix this on the W3C side... we can move this module to be "core" and localize it appropriately.

Reply:
That would be perfect. I choose this one because of simplicity.